video_xblock/static/html/student_view.html

```diff
@@ -19,7 +19,9 @@ <h3 class="hd hd-2">{{display_name}}</h3>
     {% endif %}
     {% if download_transcript_allowed and transcripts %}
         <li class="video-transcript video-download-button-custom">
-            <a  class="download-transcript" id="download-transcript-link" href="{{ transcript_download_link }}">{% trans 'Download transcript' %}</a>
+            <a  class="download-transcript"
+                id="download-transcript-link"
+                href="{{ download_transcript_handler_url }}{{ transcript_download_link }}">{% trans 'Download transcript' %}</a>
```

Comment:
Don't you need `?` after `{{download_transcript_handler_url}}`?
Like in studio_edit.html below.